_data/projects/squirrelly.yml

```diff
@@ -0,0 +1,11 @@
+name: squirrelly
+desc: Squirrelly is an ExpressJS template engine.
+site: https://nebrelbug.github.io/squirrelly/
+tags:
+- squirrelly
+- express
+- template
+- engine
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/nebrelbug/squirrelly/labels/up-for-grabs
```

Comment:
There aren't any open issues using this label - is that intentional?

Reply:
Yes, there will be some soon. 
Thanks for your time!

Reply:
There are some now